source/module_parameter/parameter.h

```diff
@@ -38,6 +41,17 @@ class Parameter
     Input_para input;
     // System parameters
     System_para sys;
+
+#ifdef __EXX
+public:
+    const Exx_Info& exx_info = exx_info_;
+    void set_exx_info() { this->exx_info_.set(this->inp); }
```

Comment:
Parameter is only a class to store parameters. 
Please remove these functions. Please move the assignment of values to parameters in the functions read_value and reset_value.